configurations/nemesis/longevity-1gb-1h-nemesis.yaml

```diff
@@ -0,0 +1,40 @@
+test_duration: 90
```

Comment:
This file seems like a duplication,

And now we have 3 identical files with different names, we should align it and leave with only one.

Reply:
Adding this file was done to reduce as much as possible the changes in NemesisJobGenerator logic.
The Job generator takes 'base-job' parameter and uses its name (e.g. 'longevity-1gb-1h') to find the base job, the base test config, etc. So we just add a new config and in the generator code we change nothing (or little), and then we are able to generate dedicated nemeses for a new backend.

Another option that was considered of how to adjust NemesisJobGenerator is to re-use 'configurations/nemesis/additional_configs/docker_backend_local.yaml' config. But it would require some more changes in JobGenerator.
Also it is less flexible in terms of parameters  - for instance at some point we can decide to use not i4i.xlarge instance type for SCTRunner on AWS, but 2xlarge, or 4xlarge, etc. This means we can also increase memory limits in 'append_scylla_args parameter', maybe increase test data volume from 1 GB to 5 GB, etc. Then it will not affect local docker backend runs experience if a separate config exists for Jenkins docker backend runs.

What do you think?

Reply:
I don't think we're gonna support multiple bases for the generator, if we'll need a certain setup we could build it, not to generate.


Regardless of the name wet should pick, we should have one copy of it